libprecompiled/Common.h

```diff
@@ -48,6 +48,10 @@ const std::string SYS_AC_ENABLENUM = "enable_num";
 enum PrecompiledError : int
 {
     // ChainGovernancePrecompiled -52099 ~ -52000
+    CODE_ACCOUNT_FROZEN = -52011,
+    CODE_ACCOUNT_AVAILABLE = -52010,
```

Comment:
```suggestion
    CODE_ACCOUNT_ALREADY_AVAILABLE = -52010,
```
add already, may be more clear?

Reply:
Done